src/chexus/validate.py

```diff
@@ -63,8 +63,13 @@ def validate(group: Group, validators: list[Validator]) -> dict[type, Validation
     tree = unroll_tree(group)
     results = {type(v): ValidationResult(v) for v in validators}
     for node in tree.values():
-        for validation in results.values():
-            validation.apply(node)
+        try:
+            for validation in results.values():
+                validation.apply(node)
+        except ModuleNotFoundError as e:
+            if e.name != 'scipp':
+                raise e
+
```

Comment:
Why do we need this?

Reply:
Because if we don't have it the `ModuleNotFoundError` will be raised when chexus is called will `--no-scipp`. We don't want that, we just want to continue with the next validator.

Reply:
Can we just remove all validators that depend on scipp, instead of going via try...except?

Reply:
We could but then we'd have to remember to register some validators as "depending on scipp", I just found the try... except better because it avoids that. The disadvantage I see with the try except version is that we have to import `scipp` in the validator where it's needed.

An advantage of doing it this way is that in the report we get an entry like
``name_of_violation_depending_on_scipp ?/5 (requires scipp)``
if there is a validator that is applicable 5 times but couldn't be run because scipp is missing.

We could of course implement that feature in some other way as well, but if we're adding extra logic anyway I don't see why the try except is worse than anything else.

Reply:
You can still list something like `Skipped validators (via --no-scipp): validator1, validator2, ...`. Knowing how many nodes it *would* have applied to does not add too much benefit?

Reply:
> Knowing how many nodes it would have applied to does not add too much benefit?

I think that particularly adds benefit because if there are many nodes where it applies then the user can estimate the value of rerunning with scipp installed. And vice versa if it applies zero times the user can be confident in the results even if `--no-scipp` was set.

Reply:
I think you are overthinking this, most users will be happy to install (or already have installed) scipp.

Reply:
Yeah you are probably right :sweat_smile: